### PR TITLE
[WFLY-19198] Move the OIDC scope related tests back to the elytron-oidc-client testsuite: delete manualmode tests

### PR DESCRIPTION
**Pull request by PrarthonaPaul**
Issue: https://issues.redhat.com/browse/WFLY-19198

____

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

### CODE REVIEW

File: testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/OidcBaseTest.java [hunk 1]
```diff
@@ -54,6 +62,7 @@
 import org.wildfly.security.jose.util.JsonSerialization;
 
 import io.restassured.RestAssured;
+import org.wildfly.test.integration.elytron.oidc.client.subsystem.SimpleServletWithScope;
```
Comment (fjuma) [inline testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/OidcBaseTest.java:65]: This should be with the org imports.

**Review by fjuma — COMMENTED** [review]
Thanks @PrarthonaPaul, I've added some small comments.

**Review by kabir — APPROVED** [review]
@PrarthonaPaul - approving the parts related to stability (i.e. the setup tasks and their usage)

File: testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/OidcBaseTest.java (hunk 1 above)
Comment (PrarthonaPaul) [reply testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/OidcBaseTest.java]: Thanks @fjuma 
I have addressed these comments.

File: testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/KeycloakConfiguration.java [hunk 2]
```diff
@@ -30,7 +32,7 @@
  */
 public class KeycloakConfiguration {
 
-    private static final String USER_ROLE = "user";
+    public static final String USER_ROLE = "user";
```
[the remark this answers is not in the document]
Comment (PrarthonaPaul) [reply testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/KeycloakConfiguration.java:35]: User role gets used [here](https://github.com/wildfly/wildfly/pull/17903/files#diff-9ab18f38769dec18e51f0a28acdbcb21340d639fd29aaacab2bd1f7d77e5ed86R30), which is why I changed it to public here.

**Comment by PrarthonaPaul** [comment]
> @PrarthonaPaul - approving the parts related to stability (i.e. the setup tasks and their usage)

Thanks @kabir

**Comment by bstansberry** [comment]
@fjuma Is this ok now?

**Review by fjuma — APPROVED** [review]
Thanks @PrarthonaPaul!

**Comment by bstansberry** [comment]
Thanks @PrarthonaPaul, @kabir and @fjuma

**Pull request merged**